cns/middlewares/swiftV2.go

```diff
@@ -30,25 +30,87 @@ const (
 	overlayGatewayV6 = "fe80::1234:5678:9abc"
 )
 
-type SWIFTv2Middleware struct {
+type K8sSWIFTv2Middleware struct {
```

Comment:
@nddq since you have renamed the middleware now, do you want to rename the file name as well to reflect it? we'll have 2 different swiftv2 middleware files under cns/middlewares

Reply:
totally forgot about this, thanks for reminding me!